src/test/scala/chiselTests/Printf.scala

```diff
@@ -8,24 +8,20 @@ import circt.stage.ChiselStage
 class SinglePrintfTester() extends Module {
   val x = 254.U
   printf("x=%x", x)
-  stop()
```

Comment:
This stop is removed so that the test can simply check if a layerblock shows up. Stops are placed under layerblocks which means that the test would pass even without the change in this PR to move printfs under layers.

FileCheck would really help here. 🙃